gaphor/diagram/diagramtools/dropzone.py

```diff
@@ -29,13 +32,10 @@ def on_motion(controller, x, y, item_class):
         parent = None
 
     if parent:
-        # create dummy adapter
-        connections = Connections()
-        adapter = Group(parent, item_class(connections))
-        if adapter and adapter.can_contain():
-            view.selection.dropzone_item = parent
-        else:
-            view.selection.dropzone_item = None
+        adapter_type = Group.registry.get_registration(type(parent), item_class)
+        # Do not do in depth check id we can actually connect.
+        # We can do that only when we have and item_class instance.
```

Comment:
This comment isn't very clear due to grammar issues. Maybe something like:

We do not do an in depth check of the id to see if we can connect, since the we do not have the item_class instance.

Reply:
Thanks for catching this.